compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -1242,9 +1242,13 @@ trait ParallelTesting extends RunnerOrchestration { self =>
 
     val (dirs, files) = compilationTargets(sourceDir, fileFilter)
 
+    val doSequentialSeparateCompilation = !flags.options.contains("-Ytest-pickler")
+    def compileDir(dir: JFile): Boolean = {
```

Comment:
Rename to `ignoreDir`? This name suggests to me that the method is compiling the directory

Reply:
Changed